backend/src/controllers/MyUserController.ts

```diff
@@ -39,7 +39,21 @@ const updateCurrentUser = async (req: Request, res: Response) => {
   }
 };
 
+const getCurrentUser = async (req: Request, res: Response) => {
+  try {
+    const currentUser = await User.findById({ _id: req.userId });
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/22#issuecomment-2380857141>#c60be5</a><span> - 09/28/2024, 06:31 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The query syntax for User.findById is incorrect. A similar issue was also found in backend/src/controllers/MyUserController.ts (line 44).
</div>
<div id="fix">
<b>Fix:</b> Modify the User.findById call to use the correct syntax by passing the userId directly
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -44 +44 @@
 -    const currentUser = await User.findById({ _id: req.userId });
 +    const currentUser = await User.findById(req.userId);
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged